README.md

```diff
@@ -96,7 +96,7 @@ After you update the source with your new proposed feature or bug fix, open a [p
 - Your PR has the name of the JIRA in the title, for example, `[KOGITO-XXX] - Awesome feature that solves it all`.
 - The PR solves only the problem described in the JIRA.
 - You have written unit tests for the particular fix or feature.
-- You ran `make test` before submitting the PR and everything is working accordingly.
+- You ran `make vet` and `make test` before submitting the PR and everything is working accordingly.
```

Comment:
can't we have make vet being executed as part of make test?

Reply:
We can, I didn't change this, I mean the purpose of make vet and make test, it was  this way earlier also

Reply:
Vet and test are different things. We could have a third command which would do both. But I would say this is for another issue ;)

Reply:
sounds good.